.gitignore

```diff
@@ -1,3 +1,5 @@
+.idea
```

Comment:
please store this in local [global gitignore](https://stackoverflow.com/a/7335487/1361089)

Reply:
done